src/types.ts

```diff
@@ -1,6 +1,9 @@
 import {ViewStyle, TextStyle} from 'react-native';
 import {MarkingProps} from './calendar/day/marking';
 
+export type ContextProp = {
```

Comment:
If you know the type, why are you using any here?

Reply:
Right. Fixed